src/System.Private.CoreLib/shared/System/Runtime/ExceptionServices/ExceptionNotification.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Runtime.ExceptionServices
```

Comment:
Same again, isn't ctor param nullable

Reply:
Technically, yes.  But based on the same reasoning as https://github.com/dotnet/coreclr/pull/23676/files#r271756174, I'd be inclined to leave it as non-nullable.